tests/System.Time.Tests/DateTimeTests.cs

```diff
@@ -17,13 +17,13 @@ public class DateTimeTests
         [Fact]
         public void CanAssignDateTimeTodayToDate()
         {
-            Date dt = DateTime.Today;
+            Date dt = (Date) DateTime.Today;
         }
```

Comment:
Should this verify that the cast did the right thing? I.e. the year, date and month match the values from DateTime.Today?

Reply:
Yes.  Rewrote these and more in two new files, `DateCastTests.cs` and `TimeCastTests.cs`.  Also changed the casting behavior. Thanks.